examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Another possibility would be to allow global style selectors within the same sheet by scoping them this way:

```javascript
{
  global: {
    html: {globalStyles},
    button: {globalStyles}
  },
  button: {localStyles}
}
```

That would be totally possible through a plugin, it appears to me like a good idea right now!